web-components/src/components/menu-overlay/MenuOverlay.ts

```diff
@@ -76,6 +80,7 @@ export namespace MenuOverlay {
     @property({ type: Boolean, attribute: "show-arrow" }) showArrow = false;
     @property({ type: Boolean }) disabled = false;
     @property({ type: String }) placement: MenuOverlay.Placement = "bottom";
+    @property({ type: Boolean}) isDatePicker = false;
```

Comment:
Why we need to add property to menu overlay component which is generic component? We should not make changes based on datepicker inside menu overlay

Reply:
Removed this